src/calendar/calendar-month.tsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState } from "react";
+import dayjs, { Dayjs } from "dayjs";
+import { MonthCell, MonthPickerContainer } from "./calendar-month.style";
+import { CalendarMonthProps, VariantMonth } from "./types";
+import { CalendarHelper } from "src/util/calendar-helper";
+
+export type Month = typeof Months[number];
```

Comment:
I will update it to typescript `type` way. since the `const Months = [..., ,...,]` is not more been use in other function.

I used const for the type as example below

`
export const Months = [
    "January",
    "February",
    "March",
    "April",
    "May",
    "June",
    "July",
    "August",
    "September",
    "October",
    "November",
    "December",
] as const;
`

Read it as 
`type Month = typeof Months[number];`

It similar as 
`
type Month = "January" |  "February", ...., ...., "December"
`